00_jit.ipynb

```diff
@@ -480,12 +458,193 @@
     }
    ],
    "source": [
-    "loaded_model = torch.jit.load(\"trace.pt\")\n",
+    "loaded_model = torch.jit.load(\"models/resnet.ts\")\n",
     "loaded_model.eval()\n",
     "x,_ = dls.one_batch()\n",
     "\n",
     "probs = loaded_model(x); probs[:3]"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Exporting the dataloader and transforms\n",
```

Comment:
Move exporting the DataLoaders to its own notebook and its own PR, there's much to adjust there and it's not jit specific. Keep each PR small and incremental - Jeremy